templates/cookie-notice.php

```diff
@@ -1,58 +1,100 @@
 <?php
 
 namespace Netzstrategen\CoreStandards;
+
+$options = [
+  [
+    'name' => 'basic',
+    'title' => __('Nur Butterkekse', Plugin::L10N),
+    'desc' => __('Erforderliche Funktionen', Plugin::L10N),
+    'atts' => [
+      'disabled' => TRUE,
+      'checked' => TRUE,
+    ],
+  ],
+  [
+    'name' => 'statistics',
+    'title' => __('Schokokekse', Plugin::L10N),
+    'desc' => __('Statistiken', Plugin::L10N),
+  ],
+  [
+    'name' => 'personalization',
+    'title' => __('Space-Cookies', Plugin::L10N),
+    'desc' => __('Personalisierung', Plugin::L10N),
+  ],
+];
+
+function create_attributes($atts) {
+  $attributes = [];
+  foreach ($atts as $name => $value) {
+    $attributes[] = $name . '="' . esc_attr($value) . '"';
+  }
+  return implode($attributes, ' ');
+}
 ?>
 
 <div class="cookie-notice" id="cookie-notice" role="banner">
   <div class="cookie-notice__body">
-    <h1>Kekse?</h1>
+    <h1><?= __('Kekse?', Plugin::L10N) ?></h1>
     <p><?= sprintf(__('Für den bestmöglichen Service bietet Ihnen diese Website Cookies. Mehr Infos finden Sie in unserer <a href="%s">Cookie-Bar</a>.', Plugin::L10N), apply_filters('core_standards/cookie_notice/policy_link', '/privacy')); ?></p>
```

Comment:
We still need to move the strings into translation files and replace them with English ones here.  It might be a good idea to move all string definitions into variables at the beginning of the file, to make editing easier? (They might be overridden by configuration in the future, which would move them to the beginning anyway.)